tests/stability/test_spill.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+
+import dask.array as da
+import pytest
+from coiled.v2 import Cluster
+from dask.distributed import Client, wait
+
+
+@pytest.mark.stability
+@pytest.mark.parametrize("keep_around", (True, False))
+def test_spilling(keep_around):
+    with Cluster(
+        name=f"test_spill-{uuid.uuid4().hex}",
+        n_workers=5,
+        worker_disk_size=55,
+        worker_vm_types=["t3.medium"],
+        wait_for_workers=True,
+        environ={"DASK_DISTRIBUTED__SCHEDULER__ALLOWED_FAILURES": "0"},
+    ) as cluster:
+        with Client(cluster) as client:
+            arr = da.random.random((200, 2**27)).persist()  # 200 GiB
+            wait(arr)
+            fut = client.compute(arr.sum())
+            if not keep_around:
+                del arr
+            wait(fut)
+
+
+@pytest.mark.stability
+def test_tensordot_stress():
+    with Cluster(
+        name=f"test_spill-{uuid.uuid4().hex}",
+        n_workers=5,
+        worker_disk_size=45,
+        worker_vm_types=["t3.medium"],
+        wait_for_workers=True,
+        environ={
+            "DASK_DISTRIBUTED__SCHEDULER__ALLOWED_FAILURES": "0",
+            "DASK_DISTRIBUTED__WORKER__CONNECTIONS__INCOMING": "1",
+            "DASK_DISTRIBUTED__WORKER__CONNECTIONS__OUTGOING": "1",
```

Comment:
We need to reduce the number of incoming and outgoing connections to avoid getting the worker `oom-killed`. With a default chunk size of 100 MiB from the `dask.array`, having 4 incoming and 4 outgoing connections at the same time means that we might use up to 800 MiB for communications. With the (previous) default of a `t3.medium` machine, this appears to be ~25 % of the actual memory that we can use. Together with the misconfigured `Worker.memory_limit` (coiled/feedback#185), we reliably trigger the `oom-killer` when running the workload without the adjusted config. See https://github.com/dask/distributed/issues/6208 for an existing upstream issue.

Reply:
@hendrikmakait is this safe to remove now that coiled/feedback#185 is resolved?

Reply:
No, there is an ongoing effort in dask/distributed#6208 to resolve the underlying issue. coiled/feedback#185 made this even worse.

Reply:
Thanks. I think we may need to skip this test for a while until this is a bit more stable (cf #280 )